catowser/catowser/Coordinators/WebContentCoordinator.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  WebContentCoordinator.swift
+//  catowser
+//
+//  Created by Andrei Ermoshin on 11/21/22.
+//  Copyright © 2022 andreiermoshin. All rights reserved.
+//
+
+import UIKit
+import CoreHttpKit
+import CoreBrowser
+
+protocol WebContentDelegate: AnyObject {
+    func didProvisionalNavigationStart()
+    func didLoadingProgressChange(_ progress: Float)
+    func didProgress(show: Bool)
```

Comment:
```suggestion
    func progressDidShow(_ show: Bool)
```
and rename other methods